components/admin_console/schema_admin_settings.jsx

```diff
@@ -383,6 +383,10 @@ export default class SchemaAdminSettings extends React.PureComponent {
 
     buildButtonSetting = (setting) => {
         const handleRequestAction = (success, error) => {
+            if (this.state.saveNeeded !== false) {
```

Comment:
nit : prefer shorter syntax
```suggestion
            if (!this.state.saveNeeded) {
```